spark-on-suse/README.md

```diff
@@ -0,0 +1,38 @@
+
+
+
+# Install a Spark Environment on Suse Enterprise Server VM
+
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2Fspark-on-suse%2Fazuredeploy.json" target="_blank">
+    <img src="http://azuredeploy.net/deploybutton.png"/>
+</a>
+<a href="http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2Fspark-on-suse%2Fazuredeploy.json" target="_blank">
+    <img src="http://armviz.io/visualizebutton.png"/>
+</a>
+
+
+## Deploying Samples
+
+To deploy a sammple using the Azure Portal, click the **Deploy to Azure** button found at the beginning of this page.
+
+To deploy the sample via the command line (using [Azure PowerShell or the Azure CLI](https://azure.microsoft.com/en-us/downloads/)) you can use the scripts.
+
+Simple execute the script and pass in the folder name of the sample you want to deploy.  For example:
+
+```PowerShell
+.\Deploy-AzureResourceGroup.ps1 -ResourceGroupLocation 'eastus' -ArtifactsStagingDirectory '[foldername]'
```

Comment:
Since this sample has artifacts that need to be staged, just provide the cmd line example that sets the switch (i.e. the second one)